src/edu/csus/ecs/pc2/imports/ccs/ContestSnakeYAMLLoader.java

```diff
@@ -440,7 +440,8 @@ public IInternalContest fromYaml(IInternalContest contest, String[] yamlLines, S
             setCcsTestMode(contest, ccsTestMode);
         }
         
-        // TODO 701 change LOAD_SAMPLE_JUDGES_DATA, false); to , true);
+        // TODO samples - At this time loading samples is not the default, there is a plan to load by default
+        // at a later date.   This would be done to avoid a configuration mistake of not loading sample data
         boolean loadSamples = fetchBooleanValue(content, LOAD_SAMPLE_JUDGES_DATA, false);
```

Comment:
did this part change with the change to the default?

Reply:
> did this part change with the change to the default?

As I understand it, making a change to this part is what "changing the default from false to true" is about.  I'm pretty sure that the change in PR #800 is precisely to change the `false` in the above call to `true` (along with deleting the preceding comments).